logging/google/cloud/logging_v2/types.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+import sys
+
+from google.api_core.protobuf_helpers import get_messages
+
+from google.api import distribution_pb2
+from google.api import http_pb2
+from google.api import label_pb2
+from google.api import metric_pb2
+from google.api import monitored_resource_pb2
+from google.cloud.logging_v2.proto import log_entry_pb2
+from google.cloud.logging_v2.proto import logging_config_pb2
+from google.cloud.logging_v2.proto import logging_metrics_pb2
+from google.cloud.logging_v2.proto import logging_pb2
+from google.logging.type import http_request_pb2
+from google.protobuf import any_pb2
+from google.protobuf import descriptor_pb2
+from google.protobuf import duration_pb2
+from google.protobuf import empty_pb2
+from google.protobuf import field_mask_pb2
+from google.protobuf import struct_pb2
+from google.protobuf import timestamp_pb2
+from google.rpc import status_pb2
+
+names = []
+for module in (
+        distribution_pb2,
+        http_pb2,
+        label_pb2,
+        metric_pb2,
+        monitored_resource_pb2,
+        log_entry_pb2,
+        logging_config_pb2,
+        logging_metrics_pb2,
+        logging_pb2,
+        http_request_pb2,
+        any_pb2,
+        descriptor_pb2,
+        duration_pb2,
+        empty_pb2,
+        field_mask_pb2,
+        struct_pb2,
+        timestamp_pb2,
+        status_pb2,
+):
+    for name, message in get_messages(module).items():
+        message.__module__ = 'google.cloud.logging_v2.types'
+        setattr(sys.modules[__name__], name, message)
+        names.append(name)
```

Comment:
Grr, #4715 still unanswered, and we are propagating the pattern.

Reply:
#soon, I promise, it's just I need to smother gax first.